packages/scene-composer/src/components/panels/AddComponentMenu.tsx

```diff
@@ -128,7 +128,7 @@ export const AddComponentMenu: React.FC<AddComponentMenuProps> = ({ onSelect })
     const component: IEntityBindingComponentInternal = {
       ref: THREE.MathUtils.generateUUID(),
       type: KnownComponentType.EntityBinding,
-      valueDataBinding: { dataBindingContext: '' },
+      valueDataBinding: { dataBindingContext: undefined },
```

Comment:
Why instantiate as `undefined`?

Reply:
string is not an expected type for dataBindingContext